### PR TITLE
Extract method for aggregated maintenance tests

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by okurz** [comment]
@asmorodskyi like this?

**Comment by asmorodskyi** [comment]
No . Need to merge this peace into my branch anyway so easier will do this in my branch . This PR will just create another headache  for me

File: products/sle/main.pm [hunk 1]
```diff
@@ -698,6 +698,17 @@ sub load_consoletests() {
     }
 }
 
+# possibility to run as part of aggregated tests
+sub run_aggregated_maintenance_tests() {
+    use List::Util qw(all);
```
Comment (asmorodskyi) [inline products/sle/main.pm]: so is it ok to use "use" in a middle of the code?

File: products/sle/main.pm [hunk 2]
```diff
@@ -698,6 +698,17 @@ sub load_consoletests() {
...
+    my %test_repos = get_env_by_pattern("\\w+_TEST_REPO");
+    my @values     = values %test_repos;
```
Comment (asmorodskyi) [inline products/sle/main.pm]: Why we have a function used in one place and return not what we actually need ?

File: products/sle/main.pm [hunk 3]
```diff
@@ -698,6 +698,17 @@ sub load_consoletests() {
     }
 }
 
+# possibility to run as part of aggregated tests
+sub run_aggregated_maintenance_tests() {
+    use List::Util qw(all);
+    my %test_repos = get_env_by_pattern("\\w+_TEST_REPO");
```
Comment (asmorodskyi) [inline products/sle/main.pm]: why we have this function in utils ? isn't utils is a storage for something WIDELY used in project ?

**Comment by asmorodskyi** [comment]
@sysrich can you please comment on questions that I raise here . Still think that we don't need to merge this PR as I can do same thing in terms of my PR regarding extratest. But your answers will give me some light how to correctly merge this code

**Comment by okurz** [comment]
your PR includes this change and does it better. so done here.

**Pull request closed without merging**